# Generated by Django 3.0.7 on 2020-06-20 10:30

import django.contrib.auth.validators
import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('myauth', '0002_auto_20200614_1934'),
    ]

    operations = [
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, max_length=150, unique=True, validators=[django.core.validators.RegexValidator('^[a-zA-Z0-9_\\-\\.]+$', 'Имя пользователя должно состоять из английских букв, цифр и -,_,.'), django.contrib.auth.validators.ASCIIUsernameValidator]),
        ),
    ]
